app/js/controllers/foyer/individuForm.js

```diff
@@ -47,9 +49,9 @@ angular.module('ddsApp').controller('FoyerIndividuFormCtrl', function($scope, op
         $scope.individu.statutMarital = 'mariage';
     }
 
-    var isIndividuParent = IndividuService.isRoleParent(options.individuRole);
+    $scope.isIndividuParent = IndividuService.isRoleParent(options.individuRole);
```

Comment:
That's a remaining of the time when we wanted to ask if the child was living at his/her parents home. Will rollback this.